src/main/java/sign_up/use_case/SignUpResponseModel.java

```diff
@@ -0,0 +1,20 @@
+package sign_up.use_case;
+
+public class SignUpResponseModel {
+    // Whether the attempt to create user was successful
+    public boolean creationSuccess;
```

Comment:
These all should be private to avoid any unintentional reassignments, etc. Make these private fields and add getters.